src/app/lightning/get-balances.ts

```diff
@@ -41,6 +41,12 @@ export const getOffChainBalance = async (): Promise<Satoshis | ApplicationError>
     },
   })
 
+export const getOffChainChannelBalances = async () => {
+  const offChainService = LndService()
+  if (offChainService instanceof Error) return offChainService
+  return offChainService.getInboundOutboundBalance()
+}
+
```

Comment:
is this function of any use at all? why not directly using `getInboundOutboundBalance()`?

Reply:
remove this function